src/main/resources/application.yaml

```diff
@@ -5,7 +5,7 @@ management.endpoints.web:
   exposure.include: info, health, metrics, prometheus
   base-path: /internal/actuator
 
-spring.profiles.active: local
+spring.profiles.active: local, mock
```

Comment:
Vi har vel ikke mulighet til å snakke med sts eller veilarbabac når man kjører lokalt, så trenger vi en egen profil for mocking?

Reply:
Mockserver skal ikke kjøres i testene - derfor i profil

Reply:
Skjønner